### PR TITLE
Remove mention of XBlocks that aren't installed by default

### PR DESCRIPTION
**Pull request by nedbat**

### CODE REVIEW

**Comment by nedbat** [comment]
@lamagnifica Sorry, I overlooked that these XBlocks are installed on edx.org, but not by default in Open edX.  We should remove them from the Dogwood release notes.

File: en_us/open_edx_release_notes/source/dogwood.rst [hunk 1]
```diff
@@ -30,30 +30,6 @@ The following new features are included in the Open edX Dogwood release.
 New XBlocks
 ==============
```
Comment (lamagnifica) [inline en_us/open_edx_release_notes/source/dogwood.rst:25]: let's retitle this New LTI XBlock and drop the subheading below

**Comment by lamagnifica** [comment]
The availability and installation of XBlocks is a pretty significant question at this time. 
Should you also be removing the documentation for these two exercise types from the opencoursestaff guide referenced in the links? doc on how to enable them for a course and set them up is pretty useless if they can't install them at the site level...
sigh.

**Comment by nedbat** [comment]
Hmm, I guess we should remove from there also.  I'm really torn about whether edx.org should be documenting these XBlocks, or whether we should be encouraging them to document them well enough that we can just refer to their docs.  Maybe we can get an opinion from @jbarciauskas on Monday.

**Comment by jbarciauskas** [comment]
Based on our conversation this morning :+1: on removing, at least while we sort things out, and potentially link to Confluence page or whatever we decide.

**Comment by nedbat** [comment]
Closing this to make a new PR against the proper branch.

**Pull request closed without merging**